measure_cocoapod_size.py

```diff
@@ -192,6 +197,14 @@ def GetPodSizeImpact(parsed_args):
     if pod_sources: ValidateSourceConfig(pod_sources)
   except ValueError as e:
     raise ValueError("could not decode JSON value %s: %s" % (parsed_args.cocoapods_source_config.name, e))
+  # Set iOS version for the project, a default version of iOS will be added if
```

Comment:
Where is the default version defined?

Reply:
It was a mistake, the comment has been updated.